deploy/docker/Dockerfile

```diff
@@ -1,19 +1,55 @@
-FROM openjdk:8u342-oracle as build-stage
+ARG FLINK_VERSION
+ARG DINKY_VERSION
+
+FROM flink:${FLINK_VERSION}-scala_2.12-java8 as flink-base
```

Comment:
Nit: if consolidation scala version, could remove scala profile from pom.xml

Reply:
> Nit: if consolidation scala version, could remove scala profile from pom.xml

I think we can have an additional discussion on whether scala2.11 is no longer needed.